docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -145,8 +145,10 @@ that is printed out.
 For the exact list of named groups ``@linter`` recognizes, see the `API
 documentation <https://api.coala.io/en/latest/>`__.
 
-For more info generally on regexes, see `Python re module
-<https://docs.python.org/3/library/re.html>`_.
+.. note::
+
+    Please refer to `Python3 re module <https://docs.python.org/3/library/re.html>`_ and `Dive into python <http://www.diveintopython3.net/regular-expressions.html>`_'s for information
```

Comment:
This doesn't fits right 😄
> Dive into python's for information

Reply:
Okay well I've tried different methods for breaking it into new line but it ignores it. Any ideas?

Reply:
I meant grammatically actually 😄 - What happens if you break the line at ```and``` (Remember, no trailing spaces)